dexmaker/src/main/java/com/google/dexmaker/stock/ProxyBuilder.java

```diff
@@ -163,8 +167,9 @@ private ProxyBuilder(Class<T> clazz) {
      * DexMaker#generateAndLoad DexMaker.generateAndLoad()} for guidance on
      * choosing a secure location for the dex cache.
      */
-    public ProxyBuilder<T> dexCache(File dexCache) {
-        this.dexCache = dexCache;
+    public ProxyBuilder<T> dexCache(File dexCacheParent) {
+        dexCache = new File(dexCacheParent, "v" + Integer.toString(VERSION));
+        dexCache.mkdirs();
```

Comment:
Should this be mkdir (without the 's')?
The previous code didn't make the dexCache directory if it didn't exist.  
This looks like a minor change in behavior.